### PR TITLE
[Enhancement] Refactor main page readme.

### PR DESCRIPTION
**Pull request by liuwenran**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmediting/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmediting/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with some of those projects.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,155 +202,271 @@ pip3 install -e .
...
 
-- ✅ [SwinIR](configs/swinir/README.md) (ICCVW'2021)
-- ✅ [NAFNet](configs/nafnet/README.md) (ECCV'2022)
-- ✅ [Restormer](configs/restormer/README.md) (CVPR'2022)
+<div align=center>
+<video src="https://user-images.githubusercontent.com/12756472/158972852-be5849aa-846b-41a8-8687-da5dee968ac7.mp4"/>
+</div>
 
-</details>
+<div align=center>
```
Comment (Z-Fran) [inline README.md:431]: This demo is Video Interpolation.

**Comment by zengyh1900** [comment]
we need a new teaser.

File: README.md [hunk 2]
```diff
@@ -319,12 +351,16 @@ If MMEditing is helpful to your research, please cite it as below.
...
+
+## 🎫 License
 
 This project is released under the [Apache 2.0 license](LICENSE).
 Please refer to [LICENSES](LICENSE) for the careful check, if you are using our code for commercial matters.
 
-## Projects in OpenMMLab 2.0
+<p align="right"><a href="#top">🔝Back to top</a></p>
+
+## 🏗️ Projects in OpenMMLab 2.0
```
Comment (zengyh1900) [inline README.md:363]:
```suggestion
## 🏗️OpenMMLab Family
```

**Review by zengyh1900 — APPROVED** [review]
Great work. We need a Chinese version (can be implemented in another PR)

**Review by Z-Fran — APPROVED** [review]

**Pull request merged**